### PR TITLE
Modify installer to support mode without tomcat

### PR DESCRIPTION
**Pull request by bbimber**
Hello,

We deploy LabKey to our cluster, where it runs in a mode without tomcat.  Years ago I wrote our deploy code to copy/paste a bunch of this script, since I was unable to use this script verbatim because we are not attempting to start/stop the tomcat service.  However, it is pretty easy to just add an argument to the official script to avoid start/stopping the tomcat server.  This PR adds that.

While extracting the LK tar and copying modules+ pipline_libs isnt hard, it would be slightly better to use the official LK script for this, if something ever changes in the future in that process.

Thank you for considering this.

### CODE REVIEW

File: distributionResources/manual-upgrade.sh [hunk 1]
```diff
@@ -70,6 +70,8 @@ print_usage()
     echo ""
     echo "        --tomcat_lk: for LabKey Internal Hosted Server Installed Only"
     echo ""
+    echo "        --skip_tomcat: the script will not attempt to start/stop tomcat.  you must do this manually before/after running the script. This might be suitable for remote pipeline servers as well.
```
Comment (labkey-susanh) [inline distributionResources/manual-upgrade.sh]:
```suggestion
    echo "        --skip_tomcat: the script will not attempt to start/stop tomcat.  You must do this manually before/after running the script.
```
The mention of a suggestion about remote pipeline servers might only serve to confuse, particularly if it is not a more definite recommendation.

**Review by labkey-susanh — APPROVED** [review]
Seems fine as minimal modification to the existing script.  One suggestion for updating the usage text, but no further review is required.

File: distributionResources/manual-upgrade.sh (hunk 1 above)
Comment (bbimber) [reply distributionResources/manual-upgrade.sh]: Sure, seems like a good idea

**Comment by bbimber** [comment]
@labkey-susanh thanks - i added your suggestion.  i am not able to merge this - do you do this, or do we need another LK reviewer?

**Pull request merged**